go/runtime/client/client.go

```diff
@@ -190,8 +190,12 @@ func (c *runtimeClient) CheckTx(ctx context.Context, request *api.CheckTxRequest
 	if err != nil {
 		return fmt.Errorf("client: failed to get light block at height %d: %w", rs.CurrentBlockHeight, err)
 	}
+	epoch, err := c.common.consensus.Beacon().GetEpoch(ctx, consensus.HeightLatest)
```

Comment:
I think the above `LightBlock` query should also be made at  `consensus.HeightLatest` here.

Reply:
Hm, but during execution it happens at the last round height to ensure consistency. It would make sense for `CheckTx` to do the same?

Reply:
oh right, yeah it makes sense as it is then.